### DISCUSSION TITLE
Maximise or resize a floating window over multiple monitors?

### DISCUSSION THREAD

**Discussion by bradwood** [i3/i3#5582] [Q&A] [answered]
Hi

I have a laptop and 2 external monitors, each of which sit above and to the left, and right, respectively, of the laptop screen.

Sometimes I have a very wide spreadsheet or kanban board that I want to be able to stretch across _both_ external monitors. I can't find a way to do this in either tiled or floating window mode. Can anyone help?

Thanks in advance!

**Comment by orestisfl** [comment, marked as answer]
The command for that is `fullscreen toggle global`

https://i3wm.org/docs/userguide.html#_toggling_fullscreen_mode_for_a_window

https://i3wm.org/docs/userguide.html#manipulating_layout

**Reply by bradwood** [reply]
it doesn't seem to do anything

**Reply by orestisfl** [reply]
This is an i3 command which you can bind to a key combination. To run it from the terminal you need to run `i3-msg fullscreen toggle global`.

**Reply by bradwood** [reply]
yeah, I've done exactly that. I use [`sxhkd`](https://github.com/baskerville/sxhkd) as a stand-alone hotkey daemon, rather than i3, so I can manage hot-keys independently of the window manager.  This works perfectly well with i3. 

I added this config to the file , which doesn't seem to actually _do_ anything.
```
super + shift + ctrl + f
  i3-msg fullscreen toggle global
```
What should I actually expect when I hit this hotkey? as when I do, nothing seems to happen.

**Reply by orestisfl** [reply]
Alright, let's get sxkhd out of the way, what happens when you run `i3-msg fullscreen toggle global` from the terminal?

**Reply by bradwood** [reply]
it works, and maximises the terminal window over _all_ my screens, which is great. Thank you. There must be some weirdness with my sxkhd. Thanks for this, I really should have tried this myself. 

Another question, if I may - is there a way to limit this behaviour to target only a _subset_ of my displays. (ie, only the external monitors, and not the laptop monitor)?

**Reply by orestisfl** [reply]
No, fullscreen over a subset of the monitors is not something this command does
